app/routes/dashboard/apps/$appId/index.tsx

```diff
@@ -57,6 +59,16 @@ export const Application = () => {
     return appIdData.relaysToday.Count.Total >= appIdData.endpoint.appLimits.dailyLimit
   }, [appIdData])
 
+  const allZeros = () => {
```

Comment:
maybe not worth adding this banner into the actual page because of your conversation with Arash, but i do think it we could merge this banner component in by itself.

Reply:
@kutoft I spoke with Gabi about this, this has been put on a feature flag now so a user has to have all zeroes for relays AND we need to turn the feature flag on to make this change.  But keeps us from showing new users this as that would look bad overall.